build/cpp/warble/inc/warble_functions.h

```diff
@@ -26,120 +26,111 @@ using warble::WarbleRequest;
-  static const Any
-  FollowReplyPackager(const std::vector<std::vector<std::string>> &result);
-
-  // receives a follow request as an any and packages it for func
-  // @return: A vector allowing for multiple calls to func
-  //  vector will have size 2
-  //  the int value 1 indicates a get request to func
-  //  the first string indicates the username_following and to_follow_following
-  //  the second string indicates the to_follow for username and username for
-  //  to_follow
-  static const std::vector<std::tuple<int, std::string, std::string>>
-  ReadRequestPackager(const Any &any);
-
-  // receives a vector containing all the warbles in reply to the requested
-  // warble
-  // @param: the vector allowing for multiple values to be returned
-  //  the 2D vector will be a placeholder as the register user reply is
-  //  indicated by the GRPC status
-  // @return: the empty follow reply
-  static const Any
-  ReadReplyPackager(const std::vector<std::vector<std::string>> &result);
-
-  // receives a profile request as an any and packages it for func
-  // @return: A vector allowing for multiple calls to func
-  //  vector will have size 3
-  //  the int value 1 indicates a get request to func
-  //  the first string indicates the username, username_following,
-  //  username_followers the second string is a placeholder and will be empty
-  static const std::vector<std::tuple<int, std::string, std::string>>
-  ProfileRequestPackager(const Any &any);
-
-  // receives a vector containing the two vectors of the user's followers and
-  // followings
-  // @param: the vector allowing for multiple values to be returned
-  //  the 2D vector will have three vectors containing the username,
-  //  username_followers, and username_followings
-  // @return: the profile reply containing the two vectors of the user's
-  // followers and followings
-  //  empty if username is not found
-  static const Any
-  ProfileReplyPackager(const std::vector<std::vector<std::string>> &result);
-
-private:
-  // the unordered map for associating function names with functions
-  //  function names map to request function and reply function
-  static const std::unordered_map<
-      std::string,
-      std::pair<std::function<std::vector<
-                    std::tuple<int, std::string, std::string>>(Any)>,
-                std::function<Any(std::vector<std::vector<std::string>>)>>>
-      func_map_;
-  // a counter for the warble id
-  static int warble_count_;
-};
+// receives a register user request as an any and packages it for func
+// @param: the any containing the register user request
+// @return: the vector allowing for multiple calls to func
+//  vector will have size 3
+//  the int 0 indicates a put request to func
+//  the first string indicates the user's username, username_following, and
+//  username_followers the second string indicates the user's username
+const std::vector<std::tuple<int, std::string, std::string>>
```

Comment:
Changed from static class methods to functions existing in the warble_functions namespace.